packages/v4/patternfly-docs/content/design-guidelines/components/menu/menu.md

```diff
@@ -97,6 +97,35 @@ When using red text for destructive actions, it is still recommended to require
 
 <img src="./img/red-delete-text.png" alt="drilldown menu" width="935"/>
 
+### Disabling menus and menu options
+Menus or menu items can be disabled for multiple reasons. The reason an action is disabled informs which design pattern you should use.
+
+1. **An action can be performed, but is not recommended.**
```

Comment:
Thinking instead of having a numbered list, maybe they should just be mini headings -with the text underneath. Because right now it looks quite weird, with the example not being indented with the list: 
<img width="885" alt="image" src="https://user-images.githubusercontent.com/51165119/222176950-7c22bd4a-93fe-4e67-9dc6-ac664e1fba40.png">